build.gradle

```diff
@@ -6,15 +6,21 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.0'
-        // Exclude the version that the android plugin depends on.
-        configurations.classpath.exclude group: "com.android.tools.external.lombok"
```

Comment:
Why is this no longer needed?

Reply:
There is no need any more for Retrolambda. Java 8 is supported out of the box since a long time